src/common/types.ts

```diff
@@ -50,6 +50,7 @@ export type CalculatedColumn<TRow, TField extends keyof TRow = keyof TRow> =
     idx: number;
     width: number;
     left: number;
+    formatter: React.ComponentType<FormatterProps<any, TRow>>;
```

Comment:
Maybe this?
```typescript
{ [key: string]: any }
```
`any` includes primary type and array, which are apparently not correct to be a `Row`. However I am not sure the above type is accurate either. Have you discussed with @nstepien about this before when you guys first worked on migrating to typescript?